### PR TITLE
study for 3-31

### PR DESCRIPTION
**Pull request by awoodrum87**
2/5 
weekend aka catch up tyme

### CODE REVIEW

**Comment by awoodrum87** [comment]
4/5
re read the readings and have a clearer grasp on this.

File: study.md [hunk 1]
```diff
@@ -29,21 +29,25 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The event loop is the the management of tasks a program executes that, when illustrated, has a loop like
+nature. the loop puts callback functions in a task que and when the task in the stack is complete, the task in the cue
+goes to the stack. When that is implemented the next task in the que gets pushed to the stack. So on and so forth..
```
Comment (jordanallain) [inline study.md:34]: Super close here. The call stack is where synchronous code waits to be ran. Asynchronous code wait in the queue. The event loop waits for the stack to be empty, then it grabs the next thing waiting in the queue and moves it to the stack to be run.

File: study.md [hunk 2]
```diff
@@ -29,21 +29,25 @@ Asynchronous - operations that may start in a certain sequence and may end in an
...
+things by using callback functions to create sort of a cue of initiated tasks, and then implement them when called.
 ```
 
 ## Async Advantages and Disadvantages
 
 What are some advantages and disadvantages of async?
 
 ```md
-<!-- your answer here -->
+code can run out of order
```
Comment (jordanallain) [inline study.md:52]: You can do better here Angie!

**Comment by jordanallain** [comment]
👍 🇯🇴

**Pull request closed without merging**